tsl/test/expected/data_node.out

```diff
@@ -1579,3 +1579,216 @@ DROP DATABASE :DN_DBNAME_3;
 DROP DATABASE :DN_DBNAME_4;
 DROP DATABASE :DN_DBNAME_5;
 DROP DATABASE :DN_DBNAME_6;
+-----------------------------------------------
+-- Test alter_data_node()
+-----------------------------------------------
+SELECT node_name, database, node_created, database_created, extension_created FROM add_data_node('data_node_1', host => 'localhost', database => :'DN_DBNAME_1');
+  node_name  |    database    | node_created | database_created | extension_created 
+-------------+----------------+--------------+------------------+-------------------
+ data_node_1 | db_data_node_1 | t            | t                | t
+(1 row)
+
+SELECT node_name, database, node_created, database_created, extension_created FROM add_data_node('data_node_2', host => 'localhost', database => :'DN_DBNAME_2');
+  node_name  |    database    | node_created | database_created | extension_created 
+-------------+----------------+--------------+------------------+-------------------
+ data_node_2 | db_data_node_2 | t            | t                | t
+(1 row)
+
+SELECT node_name, database, node_created, database_created, extension_created FROM add_data_node('data_node_3', host => 'localhost', database => :'DN_DBNAME_3');
+  node_name  |    database    | node_created | database_created | extension_created 
+-------------+----------------+--------------+------------------+-------------------
+ data_node_3 | db_data_node_3 | t            | t                | t
+(1 row)
+
+GRANT USAGE ON FOREIGN SERVER data_node_1, data_node_2, data_node_3 TO :ROLE_1;
+SET ROLE :ROLE_1;
+CREATE TABLE hyper1 (time timestamptz, location int, temp float);
+CREATE TABLE hyper2 (LIKE hyper1);
+CREATE TABLE hyper3 (LIKE hyper1);
+SELECT create_distributed_hypertable('hyper1', 'time', 'location', replication_factor=>1);
+NOTICE:  adding not-null constraint to column "time"
+ create_distributed_hypertable 
+-------------------------------
+ (10,public,hyper1,t)
+(1 row)
+
+SELECT create_distributed_hypertable('hyper2', 'time', 'location', replication_factor=>2);
+NOTICE:  adding not-null constraint to column "time"
+ create_distributed_hypertable 
+-------------------------------
+ (11,public,hyper2,t)
+(1 row)
+
+SELECT create_distributed_hypertable('hyper3', 'time', 'location', replication_factor=>3);
+NOTICE:  adding not-null constraint to column "time"
+ create_distributed_hypertable 
+-------------------------------
+ (12,public,hyper3,t)
+(1 row)
+
+INSERT INTO hyper1
+SELECT t, (abs(timestamp_hash(t::timestamp)) % 3) + 1, random() * 30
+FROM generate_series('2022-01-01 00:00:00'::timestamptz, '2022-01-05 00:00:00', '1 h') t;
+INSERT INTO hyper2 SELECT * FROM hyper1;
+INSERT INTO hyper3 SELECT * FROM hyper1;
+-- create view to see the data nodes and default data node of all
+-- chunks
+CREATE TEMPORARY VIEW chunk_default_data_node AS
```

Comment:
Does it make sense to create this view in system catalog?

Reply:
Probably in timescaledb_information or timescaledb_experimental, but let's make a separate PR in that case.